src/confluent_kafka/src/Admin.c

```diff
@@ -481,6 +482,16 @@ static PyObject *Admin_create_topics (Handle *self, PyObject *args,
                                 goto err;
                         }
                 }
+                if(!newt->replica_assignment){
+                        if((newt->replication_factor != -1 && newt->num_partitions != -1) || (newt->replication_factor == -1 && newt->num_partitions == -1)){
+                                continue;
+                        }
+                        else{
+                                i++;
+                                goto err;
```

Comment:
Recommend switch the if-statement to avoid an else block:
```c
if (..whatever is wrong) {
  i++;
  goto err;
}
```

General recommendation is to keep the level of indent blocks down and typically only do indents for the error cases, e.g.:
```c
if (some_error) {
  handle_it..
  and stuff;
}

good_things();

if (some other_error)
  goto err;

more good things();

```